duffle.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "riff",
+    "version": "0.1.0",
+    "schemaVersion": "v1.0.0-WD",
+    "description": "Installs riff",
+    "keywords": [
+        "riff"
+    ],
+    "maintainers": [
+        {
+            "name": "Swapnil Bawaskar",
+            "email": "sbawaska@pivotal.io",
+            "url": "https://projectriff.io"
+        }
+    ],
+    "invocationImages": {
+        "cnab": {
+            "name": "cnab",
+            "builder": "docker",
+            "configuration": {
+                "registry": "projectriff"
+            }
+        }
+    },
+    "credentials": {
+        "kubeconfig": {
+            "path": "/root/.kube/config"
+        }
+    },
+    "parameters": {
+        "fields": {
+            "node_port": {
```

Comment:
for the chart the template allows specifying any service type with the default being LoadBalancer. Should we use the same pattern here, or keep it locked to NodePort/not-NodePort?